### PR TITLE
[EDNA-45] Add server Jishui

### PR DESCRIPTION
**Pull request by mkaito**
Jishui will host the Demo deployment of EDNA.

Related App PR: https://github.com/serokell/edna/pull/33

### CODE REVIEW

File: lib/common/edna/server.nix [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+        };
+      } // commonOptions;
+    };
+
+    services.nginx.enable = true;
+    services.nginx.virtualHosts.edna = {
+      default = true;
+
+      serverName = with config.networking; "${hostName}.${domain}";
+      serverAliases = [ "staging.edna.serokell.team" ];
```
Comment (zhenyavinogradov) [inline lib/common/edna/server.nix:77]: It should be different for demo, shouldn't it?

File: terraform/jishui.tf [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  zone_id = aws_route53_zone.gemini_serokell_team.zone_id
+  name    = "jishui.${aws_route53_zone.gemini_serokell_team.name}"
+  type    = "A"
+  ttl     = "60"
+  records = [aws_eip.jishui.public_ip]
+}
+
+resource "aws_route53_record" "demo_edna_cname" {
+  zone_id = data.aws_route53_zone.serokell_team.zone_id
+  name    = "staging.edna.${data.aws_route53_zone.serokell_team.name}"
```
Comment (zhenyavinogradov) [inline terraform/jishui.tf:45]: Should be different here too

File: servers/jishui/default.nix [hunk 3]
```diff
@@ -0,0 +1,34 @@
+{ modulesPath, libPath, ... }:
+{
+  imports = [
+    "${modulesPath}/virtualisation/amazon-image.nix"
```
Comment (zhenyavinogradov) [inline servers/jishui/default.nix:4]: You can use `serokell-nix.nixosModules.ec2` module now, it has apropriate configuration for ec2

File: terraform/jishui.tf (hunk 2 above)
Comment (balsoft) [inline terraform/jishui.tf:45]: Shouldn't this be `demo.edna.<...>`

**Review by zhenyavinogradov — APPROVED** [review]

**Pull request merged**